scout/server/blueprints/dashboard/controllers.py

```diff
@@ -139,6 +139,31 @@ def get_dashboard_info(adapter, data={}, institute_id=None, slice_query=None):
             "count": general_sliced_info["cohort_cases"],
             "percent": general_sliced_info["cohort_cases"] / total_sliced_cases,
         },
+        {
+            "title": "Case status tag",
```

Comment:
Why not including all available tags?

Reply:
Three reasons: Some of them make little sense together with the other things in this dia, like "technical attention" or "medical attention". Some are false friends in that they require adding multiple categories to reach a full picture, like "structural": one would need to add the number of causative ordinary SVs to these to get a full picture. Plus the diagram gets rather large, and the actual table is sooo much more informative.

I considered adding properties to the `CASE_TAGS` dict, but found that a bit of overkill and thought this a decent start with the ones that were "obvious". But why not, let us just do all for now, and we can fix it from there.